dot/sync/syncer.go

```diff
@@ -136,6 +136,16 @@ func (s *Service) IsSynced() bool {
 	return s.chainSync.syncState() == tip
 }
 
+// HighestBlock gets the highest known block number
+func (s *Service) HighestBlock() int64 {
+	highestBlock, err := s.chainSync.getHighestBlock()
+	if err != nil {
+		logger.Errorf("failed to get the highest block: %s", err)
```

Comment:
maybe change to `Warnf`?

Reply:
I will search more about that and put my opinion here tomorrow, ok? 🙂

Reply:
Morning, I found some definitions about warn vs error and I believe that makes sense. I will change it to `Warnf` tonight.

Reasons:
* this error doesn't block all process
* this error just means that we are missing an information and the reason for that
* The application there is a chance to recover from this error later

Resources:
* https://reflectoring.io/logging-levels/
* https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels

Reply:
Done in a3b4324